wikimedia/wikipedia-ios
# [YiR] Metrics Part 1

**Phabricator:** https://phabricator.wikimedia.org/T376062

### Notes
This PR adds metrics to our progress so far on Year in Review. 

### Test Steps
_Note: We can lean on QA for a comprehensive check of these_
1. Set WMFLogging.h to all to display payloads in console
2. Ensure device region is set to Italy or Mexico, ensure primary app language is set to English, Spanish, or Italian.
3. Log into an account with edits on primary app language, OR build up a history tab of 6+ read articles.
4. Go to Developer Settings and enable feature flag.
5. Background / foreground to trigger report data population
6.  Click through year in review, check console event payloads against the events highlighted in green in the [deck](https://docs.google.com/presentation/d/1uAiqSlb_vpJsW2KJSM2X38JCDflRTOI2b57ygtad45I/edit#slide=id.g30986b3155d_0_276)
